package.json

```diff
@@ -57,18 +57,22 @@
 		"@typescript-eslint/eslint-plugin": "5.48.1",
 		"@typescript-eslint/parser": "5.48.1",
 		"eslint": "^8.31.0",
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20221229",
+		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20231004",
 		"eslint-plugin-simple-import-sort": "^8.0.0",
 		"formdata-polyfill": "^4.0.10",
```

Comment:
Dont mix cleanup and fixes with your actual functionality changes. Linting should be a separate pull request.